spec/spec_helper.rb

```diff
@@ -0,0 +1,25 @@
+$LOAD_PATH << File.join(File.dirname(__FILE__), '..', 'lib')
+
+require "rails"
+require "active_record"
+require "push-core"
+require "multi_json"
+require "./lib/push-apns"
+
+ActiveRecord::Base.establish_connection(:adapter => "sqlite3", :database => ":memory:")
+ActiveRecord::Schema.define(:version => 1) do
```

Comment:
This would need manually updating if any schema changes occur. Any way we could load this up from the generators (perhaps using a dummy app in `spec/dummy`)